apm-sniffer/apm-toolkit-activation/apm-toolkit-common/pom.xml

```diff
@@ -0,0 +1,102 @@
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one or more
+  ~ contributor license agreements.  See the NOTICE file distributed with
+  ~ this work for additional information regarding copyright ownership.
+  ~ The ASF licenses this file to You under the Apache License, Version 2.0
+  ~ (the "License"); you may not use this file except in compliance with
+  ~ the License.  You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>apm-toolkit-activation</artifactId>
+        <groupId>org.apache.skywalking</groupId>
+        <version>8.4.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>apm-toolkit-common</artifactId>
+
+    <properties>
+        <shade.com.google.source>com.google</shade.com.google.source>
+        <shade.com.google.target>${shade.package}.${shade.com.google.source}</shade.com.google.target>
+        <shade.io.grpc.source>io.grpc</shade.io.grpc.source>
+        <shade.io.grpc.target>${shade.package}.${shade.io.grpc.source}</shade.io.grpc.target>
+        <shade.io.netty.source>io.netty</shade.io.netty.source>
+        <shade.io.netty.target>${shade.package}.${shade.io.netty.source}</shade.io.netty.target>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.apache.skywalking</groupId>
+            <artifactId>apm-network</artifactId>
+            <version>${project.version}</version>
```

Comment:
Hi @elk-g [apm-agent-core](https://github.com/apache/skywalking/blob/9021e6791229e8dd81ffaf7b77cb9f567723d0b6/apm-sniffer/apm-agent-core/pom.xml#L58-L62) already dependency on `apm-network`, is it repeated? If I miss something, please let me know :)

Reply:
@elk-g tell me that the shade plugin depends on the direct reference lib. @arugal

Reply:
https://github.com/elk-g/skywalking/blob/c6fc54a1e38adc77e5619135586644cd0f880f7a/apm-sniffer/apm-toolkit-activation/apm-toolkit-common/pom.xml#L49-L99

Is it here? I don't see the difference with [apm-agent-core](https://github.com/apache/skywalking/blob/9021e6791229e8dd81ffaf7b77cb9f567723d0b6/apm-sniffer/apm-agent-core/pom.xml#L171-L232).

Reply:
I tried to remove this dependency,  but it reported an incompatible error,  like `io.grpc.ManagedChannel` cannot convert to `org.apache.skywalking.apm.dependencies.io.grpc.Channel`. However when I add it, everything is ok. But I don't know the specific reason.

Reply:
@elk-g  I got it, maybe can move `GRPCLogReportServiceClient` to `apm-agent-core`, what do you think? @wu-sheng @EvanLjp

Reply:
It may be caused by the `core` including the shade. It is fine to depend this directly.